### ISSUE TITLE
range attribute vs restriction element

### ISSUE THREAD

**Issue by robUx4** [ietf-wg-cellar/ebml-specification#410]
Following the 2021-05-25 [IETF meeting](https://datatracker.ietf.org/wg/cellar/meetings/), we discussed the issue of elements that have a `range` attribute but also have a `restriction` element.

In Matroska there are elements like `TrackEntry\TrackType` `TrackEntry\Video\FieldOrder` that have both elements and some missing values in the possible range. It seems when there is a `restriction` element set, the range value is redundant and confusing. We should make them mutually exclusive.

**Comment by hubblec4** [comment]
Yes, if a restriction is present then is no reason to have a range attribute, because the restriction is more precise.
